content/20.pharmaceuticals.md

```diff
@@ -0,0 +1,394 @@
+_In vitro_ analyses reported that CQ inhibited cell entry of SARS-CoV-1 [@doi:10.1186/1743-422X-2-69] and that both CQ and HCQ inhibited viral replication within cultured cells [@doi:10.1093/cid/ciaa237], leading to early hope that it might provide similar therapeutic or protective effects in patients.
+However, while the first publication on the clinical application of these compounds to the inpatient treatment of COVID-19 was very positive [@doi:10.1016/j.ijantimicag.2020.105949], it was quickly discredited [@url:https://www.isac.world/news-and-publications/official-isac-statement].
+Over the following months, extensive evidence emerged demonstrating that CQ and HCQ offered no benefits for COVID-19 patients and, in fact, carried the risk of dangerous side effects (see Appendix XX).
+The nail in the coffin came when findings from the large-scale RECOVERY trial were released on October 8, 2020.
+This study enrolled 11,197 hospitalized patients whose physicians believed it would not harm them to participate and used a randomized, open-label design to study the effects of HCQ compared to SOC at 176 hospitals in the United Kingdom [@doi:10.1056/NEJMoa2022926].
+Rates of COVID-19-related mortality did not differ between the control and HCQ arms, but patients receiving HCQ were slightly more likely to die due to cardiac events.
+Patients who received HCQ also had a longer duration of hospitalization than patients receiving usual care and were more likely to progress to mechanical ventilation or death (as a combined outcome).
+As a result, enrollment in the HCQ arm of the RECOVERY trial was terminated early [@url:https://www.recoverytrial.net/news/statement-from-the-chief-investigators-of-the-randomised-evaluation-of-covid-19-therapy-recovery-trial-on-hydroxychloroquine-5-june-2020-no-clinical-benefit-from-use-of-hydroxychloroquine-in-hospitalised-patients-with-covid-19].
+The story of CQ/HCQ therefore illustrates how initial promising _in vitro_ analyses can fail to translate to a useful clinical approach, even for a drug that has been on the market for many years.
+
+While CQ/HCQ are well-known medications that have long been prescribed in certain contexts, investigation of another well-established type of pharmaceutical was facilitated by the fact that it was already being taken by a large number of COVID-19 patients.
+Angiotensin-converting enzyme (ACE) inhibitors and angiotensin II receptor blockers (ARBs) are among today's most commonly prescribed medications, often being used to control blood pressure [@url:https://clincalc.com/DrugStats/Drugs/Lisinopril; @doi:10.1056/NEJMp1901657].
+In the United States, for example, they are prescribed well over 100,000,000 times annually [@doi:10.18553/jmcp.2010.16.9.671].
+<!-- To Do: check ARBs/ACEIs and SARS/MERS and add a sentence if that was explored-->
```

Comment:
MERS has a different receptor, but SARS is ACE2. I didn't find much other than a brief mention of "should be further researched" a lot when it came to ace inhibitors and SARS

Reply:
There is this in vitro/molecular docking study study which investigated inhibiting ACE2 (https://doi.org/10.1161/01.HYP.0000146120.29648.36). And this 2003 perspective fleetingly hypothesized that existing ace2 inhibitors were unlikely to do much (https://doi.org/10.1016/S0092-8674(03)00976-0)

Reply:
Thank you so much!